api/config.py

```diff
@@ -47,6 +47,9 @@
         'db_server_selection_timeout': '3000',
         'data_path': os.path.join(os.path.dirname(__file__), '../persistent/data'),
     },
+    'elasticsearch': {
+        'es_endpoint': 'flywheel-elasticsearch:9200'
```

Comment:
@gsfr Has expressed desire before to keep the word `flywheel` out of this code base :)
I personally don't mind either way, since it's only a default. Up to ya'll.